connect/api/src/main/java/org/apache/kafka/connect/source/SourceTask.java

```diff
@@ -93,8 +93,8 @@ public void commit() throws InterruptedException {
      * also called when a record is filtered by a transformation, and thus will never be ACK'd by a broker.
      * </p>
      * <p>
-     * This is an alias for {@link commitRecord(SourceRecord, RecordMetadata)} for backwards compatibility. The default
-     * implementation of {@link commitRecord(SourceRecord, RecordMetadata)} just calls this method. It is not necessary
+     * This is an alias for {@link #commitRecord(SourceRecord, RecordMetadata)} for backwards compatibility. The default
```

Comment:
This has been fixed by: https://github.com/apache/kafka/pull/8291 
Please omit and rebase to get the latest changes.